_locales/en/messages.json

```diff
@@ -1936,7 +1936,7 @@
 				"content": "</a>"
 			},
 			"link_start": {
-				"content": "<br><a target=\"_blank\" rel=\"noopener noreferrer\" href=\"https://www.ghostery.com/products/plus/\">"
+				"content": "<br><a target=\"_blank\" rel=\"noopener noreferrer\" href=\"https://www.ghostery.com/products/plus/?utm_source=gbe&utm_campaign=in_app_hstats_lm\">"
 			}
```

Comment:
The changed content in this string makes sense, but I'm just confused as to where it is being used as I was unable to find anywhere that it is referenced in the project.